v2/futures/position_service.go

```diff
@@ -211,3 +211,66 @@ func (s *GetPositionModeService) Do(ctx context.Context, opts ...RequestOption)
 	}
 	return res, nil
 }
+
+// ChangeMultiAssetModeService change user's multi-asset mode
+type ChangeMultiAssetModeService struct {
+	c                 *Client
+	multiAssetsMargin string
```

Comment:
```suggestion
	multiAssetsMargin bool
```

Reply:
I referred https://binance-docs.github.io/apidocs/futures/en/#change-multi-assets-mode-trade, where it is seen as 
`POST /fapi/v1/multiAssetsMargin (HMAC SHA256)` has the parameter as below.
multiAssetsMargin | STRING | YES | "true": Multi-Assets Mode; "false": Single-Asset Mode

I follow the Binance api doc. When getting multi-asset mode, the response is bool type, but when we change by POST, its parameter multi-asset mode is string type.